tests/by-util/test_more.rs

```diff
@@ -17,6 +17,9 @@ fn test_valid_arg() {
 
         new_ucmd!().arg("-p").succeeds();
         new_ucmd!().arg("--clean-print").succeeds();
+
+        new_ucmd!().arg("-s").succeeds();
+        new_ucmd!().arg("--squeeze").succeeds();
```

Comment:
would it be possible to do it on a file ?

The coverage isn't happy

Reply:
I am not sure what you mean

When I add this : 
```rust
let scene = TestScenario::new(util_name!());
let at = &scene.fixtures;
at.touch("file.txt");

scene.ucmd().arg("-s").arg("file.txt").succeeds().no_stderr();
```
and run grcov I obtain the exact same result